Sloth.Core/Jobs/AggieEnterpriseJournalJob.cs

```diff
@@ -163,6 +169,19 @@ public async Task<AggieEnterpriseJournalJobDetails> UploadTransactions(ILogger l
 
                         jobDetails.TransactionRunStatuses.Add(transactionRunStatus);
                     }
+
+                    if (hasRejectedTransactions)
+                    {
+                        if (!await _notificationService.Notify(Notification
```

Comment:
I'd wrap wither this or the notify method in a try catch.
I don't think we want an email failure breaking the rest of the processing.

Reply:
NM, I see it is only argument exceptions. Actual send is wrapped